### PR TITLE
add transition and rotate mixins

### PR DESCRIPTION
**Pull request by luuptekman**

### CODE REVIEW

**Comment by luuptekman** [comment]
add also new function to UTILS

File: library/classes/Utils.php [hunk 1]
```diff
@@ -220,4 +220,24 @@ public function get_first_paragraph( $text ) {
...
+	 *
+	 * @return false|string
+	 */
+	public function getPostImage( $size = 'full', $postId = null ) {
+		$featuredImageUrl = get_the_post_thumbnail_url( $postId, $size );
+
+		if($featuredImageUrl) {
+			return $featuredImageUrl;
+		} else {
+			$image_id = isset( get_option( 'talvisota_general_options' )['talvisota_default_image_id'] ) ? get_option( 'talvisota_general_options' )['talvisota_default_image_id'] : null;
```
Comment (villeristi) [inline library/classes/Utils.php]: Use correct option-prefix (`digia_wp_base_`) instead of project-specific one.

Comment (villeristi) [inline library/classes/Utils.php]: You should omit the logic inside `else`-statement & use `return $this->get_default_image($size)`. Also, in WordPress one should `snake_cased_function_names` as defined in coding-standards.

**Comment by luuptekman** [comment]
changes done..

**Pull request merged**